codegra_fs/cgfs.py

```diff
@@ -1416,8 +1416,25 @@ def strippath(self, path: str) -> str:
     def load_courses(self) -> None:
         assert cgapi is not None
 
-        for course in cgapi.get_courses():
+        courses = cgapi.get_courses()
+        for dups in codegra_fs.utils.find_all_dups(
+            courses, lambda x: x['name']
+        ):
+            end = 4
+            if len(dups) != len(set(d['created_at'][:4] for d in dups)):
+                end = None
```

Comment:
Maybe we should still remove the time portion of the name? `end = 10`?

Reply:
Yeah that is probably a good idea.

Reply:
No that is not possible here as we detected that the created date of these duplicates is very close to each other. We should do this however for other cases.